grodecoder.py

```diff
@@ -1090,9 +1090,8 @@ def is_lipid(
         if "formula_no_h" in dict_count.keys():
             formula_graph = dict_count["formula_no_h"]
             selected_row = lipid_csml_charmm_gui.loc[
-                (lipid_csml_charmm_gui["Alias"] == res_name_graph)
-                & (lipid_csml_charmm_gui["Formula"] == formula_graph)
-            ]
+                (lipid_csml_charmm_gui["Alias"].apply(lambda x: res_name_graph in x)) 
```

Comment:
- Could you provide some examples as comments?
- `.apply(lambda x: res_name_graph in x)` looks a bit overengineering. Could you try something like `lipid_csml_charmm_gui["Alias"].str.contains(res_name_graph)` or `lipid_csml_charmm_gui["Alias"].str.startswith(res_name_graph)`